src/util/FactorUtil.js

```diff
@@ -346,6 +346,13 @@ function (Okta, TimeUtil) {
     return localizedQuestion.indexOf('L10N_ERROR') < 0 ? localizedQuestion : questionObj.questionText;
   };
 
+  fn.removeRequirementsFromError = function (responseJSON, policy) {
+    var passwordRequirementsAsString = this.getPasswordComplexityDescription(policy);
+    responseJSON.errorCauses[0].errorSummary = responseJSON.errorCauses[0].errorSummary
```

Comment:
better to safe check `errorCauses` is Array and `errorCause[0].errorSummary` is string.